packages/@lwc/integration-karma/test/mixed-shadow-mode/LightningElement.shadowSupportMode/index.spec.js

```diff
@@ -15,7 +16,14 @@ describe('shadowSupportMode static property', () => {
     it('should not throw for valid values', () => {
         expect(() => {
             createElement('x-valid', { is: Valid });
-        }).not.toThrowError();
+        }).not.toLogErrorDev();
+    });
+
+    // TODO [#3971]: Completely remove shadowSupportMode "any"
+    it('should warn for deprecated value "any"', () => {
+        expect(() => {
+            createElement('x-any', { is: Any });
+        }).toLogWarningDev(/Invalid value 'any' for static property shadowSupportMode/);
```

Comment:
Can we also test that `any` renders in synthetic mode, not native mode?

Reply:
Sorry, I'm not sure I understand. This test runs for both synthetic and native right?

Reply:
Right, sorry. I just mean that we should confirm that `'any'` does _not_ do the same thing as `'native'`. It should be a no-op.